meson.build

```diff
@@ -1,5 +1,6 @@
-project('dxvk', ['c', 'cpp'], version : 'v2.3.1', meson_version : '>= 0.58', default_options : [ 'cpp_std=c++17', 'warning_level=2' ])
+project('dxvk', ['c', 'cpp'], version : '2.3.1', meson_version : '>= 0.58', default_options : [ 'cpp_std=c++17', 'warning_level=2' ])
```

Comment:
You already .strip the 'v' later on. This shouldn't need to change.

Reply:
Good catch, dropped that commit from the changeset.